packages/eslint-config-airbnb-base/rules/style.js

```diff
@@ -3,531 +3,823 @@ module.exports = {
-      allowAllPropertiesOnSameLine: true,
-    }],
+    // KEEP, this necessitates further discussion. Per Mike's comments on the ES6 style guide, this rule is going to cover the standard
+    // for not only object expressions, but also for reproducing our TypeScript Hero standard for import formatting.
+    // I'd like to cover that in a separate pull request
+    'object-property-newline': [
+      'error',
+      {
+        allowAllPropertiesOnSameLine: true,
+      },
+    ],
 
     // allow just one var statement per function
+    // DISABLE, not applicable
     'one-var': ['error', 'never'],
 
     // require a newline around variable declaration
     // https://eslint.org/docs/rules/one-var-declaration-per-line
+    // DISABLE, not applicable
     'one-var-declaration-per-line': ['error', 'always'],
 
     // require assignment operator shorthand where possible or prohibit it entirely
     // https://eslint.org/docs/rules/operator-assignment
+    // SITUATIONAL ANALYSIS NEEDED; Need team input on what they prefer
     'operator-assignment': ['error', 'always'],
 
     // Requires operator at the beginning of the line in multiline statements
     // https://eslint.org/docs/rules/operator-linebreak
+    // SITUATIONAL ANALYSIS NEEDED: Need team input.
+    // This partners with the above rule, where if we decie if we want operators in multiline statements to be the delimiting character for a new line,
+    // this rule decides where we want them to be
     'operator-linebreak': ['error', 'before', { overrides: { '=': 'none' } }],
 
     // disallow padding within blocks
-    'padded-blocks': ['error', {
-      blocks: 'never',
-      classes: 'never',
-      switches: 'never',
-    }, {
-      allowSingleLineBlocks: true,
-    }],
+    // DISABLE, adds unnecessary whitespace
+    'padded-blocks': [
+      'error',
+      {
+        blocks: 'never',
+        classes: 'never',
+        switches: 'never',
+      },
+      {
+        allowSingleLineBlocks: true,
+      },
+    ],
 
     // Require or disallow padding lines between statements
     // https://eslint.org/docs/rules/padding-line-between-statements
+    // KEEP OFF
     'padding-line-between-statements': 'off',
 
     // Disallow the use of Math.pow in favor of the ** operator
     // https://eslint.org/docs/rules/prefer-exponentiation-operator
     // TODO: enable, semver-major when eslint 5 is dropped
+    // KEEP OFF
     'prefer-exponentiation-operator': 'off',
 
     // Prefer use of an object spread over Object.assign
     // https://eslint.org/docs/rules/prefer-object-spread
+    // KEEP
     'prefer-object-spread': 'error',
 
     // require quotes around object literal property names
     // https://eslint.org/docs/rules/quote-props.html
-    'quote-props': ['error', 'as-needed', { keywords: false, unnecessary: true, numbers: false }],
+    // DISABLE, can be managed by Prettier configuration: quoteProps: 'asNeeded'
+    'quote-props': [
```

Comment:
switch to `off`, can be managed by Prettier's `quoteProps` rule: `quoteProps: 'asNeeded'`